src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1440,6 +1440,15 @@ private AuthenticationResult localFlow(CallbackHandler callbackHandle,
             } catch (AuthenticationException authenticationException) {
                 callbackHandle.onError(authenticationException);
                 return null;
+            } catch (HttpResponseException httpResponseException) {
```

Comment:
I don't see the point of creating this exception...you was able to get all needed info from result object